docs/source/configuration.rst

```diff
@@ -3,3 +3,30 @@
 ****************
 Configuring Ibis
 ****************
+
+
+Working with secure clusters (Kerberos)
+---------------------------------------
+
+Ibis is compatible with Hadoop clusters that are secured with Kerberos (as well
+as SSL and LDAP).  Just like the Impala shell and ODBC/JDBC connectors, Ibis
+connects to Impala through the HiveServer2 interface (using the impyla client).
+Therefore, the connection semantics are similar to the other access methods for
+working with secure clusters.
+
+Specifically, after authenticating yourself against Kerberos (e.g., by issuing
+the appropriate ``kinit`` commmand), simply pass ``use_kerberos=True`` (and set
+``kerberos_service_name`` if necessary) to the ``ibis.impala_connect(...)``
+method when instantiating an ``ImpalaConnection``.  This method also takes
+arguments to configure LDAP (``use_ldap``, ``ldap_user``, and
+``ldap_password``) and SSL (``use_ssl``, ``ca_cert``).  See the documentation
+for the Impala shell for more details.
```

Comment:
meh.  It's literally the same except for `use_kerberos=True`.  This'll only increase the places we need to make changes.  I instead added a note to the main getting-started doc.